mmj2jar/LinuxMMJ2PATutorial.command

```diff
@@ -0,0 +1 @@
+java -XX:+UseG1GC -Xms128M -Xmx1024M -jar /users/Userone/mmj2jar/mmj2.jar "LinuxRunParmsPATutorial.txt" Y "/users/Userone/mmj2jar/" "/users/Userone/metamath/" "" 

```

Comment:
Yeah, just copied from other files -- I assume everyone has to edit these files anyway. "~" would be closer to reality.